Client/mods/deathmatch/logic/lua/CLuaManager.h

```diff
@@ -33,8 +33,7 @@ class CLuaManager
     void      OnLuaMainOpenVM(CLuaMain* pLuaMain, lua_State* luaVM);
     void      OnLuaMainCloseVM(CLuaMain* pLuaMain, lua_State* luaVM);
 
-    std::list<CLuaMain*>::const_iterator IterBegin() { return m_virtualMachines.begin(); };
-    std::list<CLuaMain*>::const_iterator IterEnd() { return m_virtualMachines.end(); };
+    auto GetLuaMains() const { return m_VirtualMachines; }
```

Comment:
Why did you pick a return `auto` here? Seems like an unusual place to use it, especially since `m_VirtualMachines` doesn't have a particularly difficult type to write. I'm happy to keep `auto` if you think we should keep it, though.

Reply:
Actually I shouldve returned `const auto&`.
The reason is that it doesnt matter what the type of `m_VirtualMachines` is. Its known to be an STL container, and I find this easier to read and maintain than:
```cpp
const std::vector<CLuaMain*>& GetLuaMains() const { return m_VirtualMachines; }
```

Reply:
Okay